SignalUtilitiesKit/Utilities/UIViewController+Utilities.swift

```diff
@@ -28,7 +28,7 @@ public final class ViewControllerUtilities : NSObject {
         }
         // Set up back button
         if hasCustomBackButton {
-            let backButton = UIBarButtonItem(title: "Back", style: .plain, target: nil, action: nil)
+            let backButton = UIBarButtonItem(title: "", style: .plain, target: nil, action: nil)
```

Comment:
This is interesting. Seems like we really have a lot of redundant code. Could you clean up a bit on all these code related to back button? Cheers!

Reply:
Yea this is a bit of a messy one - I think the main difference is when view controllers don't inherit from `BaseVC` they use this method (ie. cases where things inherit from `OWSTableViewController`/`OWSViewController`/`UICollectionViewController`) - seems to be used in ~7 places and I'm not sure we want to refactor those screens to use `BaseVC` as part of this PR

Reply:
OK, no worries! Let's leave this cleaning up for future us!